pdsfile/__init__.py

```diff
@@ -882,28 +605,266 @@ def new_pdsfile(self, key=None, copypath=False):
             this.checksums_      = self.checksums_
             this.archives_       = self.archives_
             this.voltype_        = self.voltype_
-            this.volset_         = self.volset_
-            this.volset          = self.volset
+            this.bundleset_      = self.bundleset_
+            this.bundleset       = self.bundleset
             this.suffix          = self.suffix
             this.version_message = self.version_message
             this.version_rank    = self.version_rank
             this.version_id      = self.version_id
-            this.volname_        = self.volname_
-            this.volname         = self.volname
+            this.bundlename_     = self.bundlename_
+            this.bundlename      = self.bundlename
             this.interior        = self.interior
 
+            this.volset_  = self.bundleset_
```

Comment:
Why is this in here when the properties are only needed for PDS3? Maybe we need a method in Pds3File that updates these attributes so you don't have this code copied in multiple places. (see also below)

Reply:
removed